dashboard/src/components/relocation/RetryConnectionBanner.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { withApollo } from "react-apollo";
+import Button from "@material-ui/core/Button";
+import Banner from "./Banner";
+import retryLocalNetwork from "../../mutations/retryLocalNetwork";
+
+class RetryConnectionBanner extends React.PureComponent {
+  static propTypes = { client: PropTypes.object.isRequired };
+
+  retryConnection = () => {
+    console.log("this is running");
+    retryLocalNetwork(this.props.client);
+  };
+
+  render() {
+    return (
+      <Banner
+        message="You've lost network connection."
```

Comment:
We could potentially provide a better explanation here. Maybe a link to a help document if one exists (would probably have to create that).

There's even an opportunity to use the `NetworkInformation` API (https://developer.mozilla.org/en-US/docs/Web/API/NetworkInformation) which @jamesdphillips was interested in.